randomUsernameAndPassword/assignment1/Q1.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+
+def is_anagram(str1, str2, case_sensitive):
+    
+    if not case_sensitive:
+        str1.lower()
+        str2.lower()
+    
+    str1 = str1.split(" ")
+    str2 = str2.split(" ")
+    str1 = sorted([''.join(sorted(i)) for i in str1])
```

Comment:
Short and powerful: very cool/

In this code you duplicate the logic to "normalize" a sentence: do you want to create a function that takes the sentence and return a structure that is comparable?

Reply:
Fixed